lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -754,16 +757,18 @@ def test_coefficients_gaussian_realizations_no_statsmodels(self):
         expected values, and the coefficient names also match
         expected values for a Gaussian distribution where the
         realizations are used as the predictor of the mean."""
-        data = np.array([0.0450917,  123.35383, -0.943704, -0.747887,
-                         -0.0180841, -0.655514], dtype=np.float32)
+        data = np.array([1.08333e+00, 1.09167e+00, 3.61111e-04,
```

Comment:
Likewise, we might want to explain that we are not converging to the solution in these tests because the iterator is prone to escalating small errors that results in an overly sensitive test (e.g. to package versions, processor optimisation etc). You could stick the various comments in the doc-string at the top of the file if that is tidier as it will require a bit of explanation and happens in a few places.